libs/island-ui/core/src/lib/Typography/Typography.tsx

```diff
@@ -47,41 +48,47 @@ export const Typography = ({
   links,
   paddingTop,
   paddingBottom,
+  paddingY,
```

Comment:
if we are only supporting padding or margin I would suggest only support margin rather then padding for text driven components